From 80115fef5b70e50d8aaefc84296a5fd8338b254f Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 15 Feb 2024 18:34:56 +0100
Subject: [PATCH 0600/1695] dts: stm32: describe wake up parent for pmic2 on
 stm32mp257f-dk board

Reference PWR as wakeup-parent for PMIC on this board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 087219f4948d87d20e487f08a40cb7d24e0cbff7 dts: stm32: describe wake up parent for pmic2 on stm32mp257f-dk board]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I83e0a88ee968d183b4964fba3d81d6a682730bac
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index 3c581d26c..741eed10f 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -63,6 +63,7 @@
 		compatible = "st,stpmic2";
 		reg = <0x33>;
 		st,wakeup-pin-number = <1>;
+		wakeup-parent = <&pwr>;
 		status = "okay";
 
 		regulators {
-- 
2.47.2

